pages/1.13/installing/production/system-requirements/partitions/index.md

```diff
@@ -0,0 +1,50 @@
+- Navstar Overlay replicated log: /var/lib/dcos/mesos/master/overlay_replicated_log
+- CockroachDB distributed database /var/lib/dcos/cockroach
+- - Navstar Mnesia distributed database: /var/lib/dcos/navstar/mnesia
+- Navstar Lashup distributed database: /var/lib/dcos/navstar/lashup
+- Secrets vault: /var/lib/dcos/secrets/vault
+- Exhibitor Zookeeper distributed database: /var/lib/dcos/exhibitor/zookeeper
+- History Service cache: /var/lib/dcos/dcos-history
+
+## Agent nodes
+On the agent nodes, you should use separate partitions for the following directories under `/var/lib/mesos`:
+
+- `/var/lib/mesos` - You should always host the `/var/lib/mesos` directory on a separate partition. Keep in mind that the disk space that Apache Mesos advertises in its UI is the sum of the space provided by the file system(s) underpinning the `/var/lib/mesos` directory, including any MOUNT volumes (`/dcos/volume<n>`).
+
+- `/var/lib/mesos/slave/slaves` - This directory hosts the sandbox directories for tasks. You should use a separate partition for this directory, if possible.
+
+- `/var/lib/mesos/slave/volumes` - This directory is used by frameworks that consume ROOT persistent volumes. You should use a separate partition for this directory, if possible.
+
+- `/var/lib/mesos/docker/store` - This directory stores Docker image layers that are used to provision UCR containers. You should use a separate partition for this directory, if possible.
+
+In most cases, agents do not require a separate partition for the persistent configuration override files stored in the `/var/lib/dcos` directory. You should, however, consider using separate partitions for the following directories on agent nodes:
+- `/var/lib/dcos/mesos-slave-common`
+- `/var/lib/dcos/mesos-resources`
+
+<p class="message--note"><strong>NOTE: </strong>For overall cluster health, keep in mind that the `/var/lib/mesos/slave/meta/resources`, `/var/lib/mesos/slave/volumes` and `/dcos/volume<N>` MOUNT directories must be available (or restored from backups) for reservations to be re-advertised to the frameworks. Providing separate partitions to ensure the availability of these directories and the resource offers allows operations and tasks to recover successfully.
+
+## Other directories and partitions
+In addition to the master- and agent-specific partitions, you should consider using a separate partition for each of the following directories:
+
+- `/var/lib/docker` - This directory stores Docker image layers and is used by containers launched with the Docker engine.
+
+- `/dcos/volume<n>` - These volume-identified directories--for example, `/dcos/volume0` and `/dcos/volume1`--are used by frameworks that consume MOUNT persistent volumes.
+
+- `/opt/mesosphere` - This directory contains the DC/OS binaries, libraries, and cluster configuration. If the DC/OS cluster is version 1.11, or newer, you should place this directory in its own partition
```

Comment:
If we want people to have e.g. /opt/mesosphere on dedicated partitions we should also add size recommendations, to prevent issues in the future. Because e.g. /opt/mesosphere is increasing with every DC/OS upgrade, since there is no automatic clean-up of old pkgpanda files or you need to run unsupported scripts to manually clean-up

Reply:
Took a stab at revising this to:
... Because the size of this directory tends to increase with each DC/OS upgrade and there is no automatic clean-up of old files, you should monitor disk space usage for the `/opt/mesosphere` directory carefully. In addition to monitoring the directory, you might want to create a custom script to periodically remove unused or out-of-date files.
...